lib/duo_api.rb

```diff
@@ -27,6 +27,12 @@ def initialize(ikey, skey, host, proxy = nil, ca_file: nil)
     end
     @ca_file = ca_file ||
                File.join(File.dirname(__FILE__), '..', 'ca_certs.pem')
+
+    # Constants for handling rate limit backoff
```

Comment:
These look like good candidates for constants but they are being declared as instance variables.

Can you move them out of the initialize method and into the class? Then you'd just need to get rid of the @ and you should be good to go!